### PR TITLE
Project fixes

### PR DESCRIPTION
**Pull request by jonaengs**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,89 +1,90 @@
...
+
+    version = '1.0'
+    ext {
+        appName = "minigolf-with-friends"
+        gdxVersion = '1.9.13'
+        roboVMVersion = '2.3.12'
+        box2DLightsVersion = '1.5'
+        ashleyVersion = '1.7.3'
+        aiVersion = '1.8.2'
+        junitVersion = '4.12'
```
Comment (jonaengs) [inline build.gradle:30]: add junitversion

File: build.gradle [hunk 2]
```diff
@@ -1,89 +1,90 @@
...
+
+project(":core") {
+    apply plugin: "java-library"
+
+
+    dependencies {
+        api "com.badlogicgames.gdx:gdx:$gdxVersion"
+        api "com.badlogicgames.gdx:gdx-box2d:$gdxVersion"
+        api "com.badlogicgames.ashley:ashley:$ashleyVersion"
+        api "junit:junit:$junitVersion"
```
Comment (jonaengs) [inline build.gradle:88]: add junit

File: core/src/test/java/TestTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (jonaengs) [inline core/src/test/java/TestTest.java:1]: Directory path suggested by intellij. Seems to be gradle's default test path.

File: README.md [hunk 4]
```diff
@@ -1,2 +0,0 @@
-# MinigolfWithFriends
```
Comment (jonaengs) [inline README.md:1]: These have been causing me way too much pain. Better to delete both and then add a single one later if need be.

**Review by runarsae — APPROVED** [review]
This looks good!

**Review by AleksanderWK — APPROVED** [review]
Looks good

**Pull request merged**